test/e2e/util_gatewayapi_test.go

```diff
@@ -529,8 +547,8 @@ func assertGatewayClassSuccessful(t *testing.T, name string) (*gatewayapiv1.Gate
 	nsName := types.NamespacedName{Namespace: "", Name: name}
 	recordedConditionMsg := "not found"
 
-	// Wait up to 2 minutes for the gateway class to be Accepted.
-	err := wait.PollUntilContextTimeout(context.Background(), 2*time.Second, 2*time.Minute, false, func(context context.Context) (bool, error) {
+	// Wait up to 4 minutes for the gateway class to be Accepted.
```

Comment:
Any explanation for why creating everything takes so much longer?  Should we be waiting for something else to happen first?

Reply:
Is it at all possible that adding another clause to the CEL validation is causing increasing times?  If so, we wouldn't want to use a longer CEL validation.

Reply:
> Is it at all possible that adding another clause to the CEL validation is causing increasing times? If so, we wouldn't want to use a longer CEL validation.

Well, yes, longer CEL expressions may slow down authorization process. However, the VAP we use is relatively simple and should not harm the elapsed time of tests, at least not to the degree of dozens of seconds. Also, the failures I saw were related to the fact that gatewayclass or other objects were not accepted. Which suggests that the custom resource objects were created which in turn implies that CRD existed.

I suspect the install plan approvals to be the cause of the failures. Slack thread which discusses this: [link](https://redhat-internal.slack.com/archives/CBWMXQJKD/p1742421080814679).